plugins/modules/ecs_taskdefinition.py

```diff
@@ -190,7 +192,7 @@
             linuxParameters:
                 description: Linux-specific modifications that are applied to the container, such as Linux kernel capabilities.
                 required: False
-                type: list
+                type: dict
```

Comment:
This looks like it's going from a list of dicts to a dict, is this not a breaking change?  (Although I guess we can do this with 2.0.0

Reply:
(_Imho_) it never worked as a `list`. I guess it was one of the transformation errors when `elements` where introduced.